homeassistant/components/nibe_heatpump/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "ip_address": "Remote IP address",
+          "remote_read_port": "Remote read port",
+          "remote_write_port": "Remote write port",
+          "listening_port": "Local listening port"
+        }
+      }
+    },
+    "error": {
+      "write": "Error on write request to pump. Verify your `Remote write port` or `Remote IP address`.",
+      "read": "Error on read request from pump. Verify your `Remote read port` or `Remote IP address`.",
+      "address": "Invalid remote IP address specified. Address must be a IPV4 address.",
+      "address_in_use": "The selected listening port is already in use on this system.",
+      "model": "The model selected doesn't seem to support modbus40",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
When is `already_configured` used? I don't see where we set a unique id or check otherwise for already configured entry.

Reply:
hmm. No it isn't was part of template. Let me check if i there is a serial i can use. Would be nice.

Reply:
Don't seem to be, so dropped: https://github.com/home-assistant/core/pull/78931/commits/bc1e94b8a7b904a9928576785c7ebe87be2fa53a